logging/src/jvmMain/kotlin/io/spine/logging/jvm/LogContext.kt

```diff
@@ -438,24 +410,6 @@ protected constructor(
      */
     private fun logImpl(message: String?, vararg args: Any?) {
```

Comment:
Remove the `args` parameter because it does not make sense since `message` is never a formatting template any more.